v4-client-js/src/clients/modules/get.ts

```diff
@@ -559,6 +560,56 @@ export class Get {
     return VaultModule.QueryMegavaultWithdrawalInfoResponse.decode(data);
   }
 
+  async getAffiliateInfo(address: string): Promise<AffiliateModule.AffiliateInfoResponse> {
+    const requestData = Uint8Array.from(
+      AffiliateModule.AffiliateInfoRequest.encode({
+        address,
+      }).finish(),
+    );
+
+    const data = await this.sendQuery('/dydxprotocol.affiliates.Query/AffiliateInfo', requestData);
+
+    return AffiliateModule.AffiliateInfoResponse.decode(data);
+  }
+
+  async getReferredBy(address: string): Promise<AffiliateModule.ReferredByRequest> {
+    const requestData = Uint8Array.from(
+      AffiliateModule.ReferredByRequest.encode({
+        address,
+      }).finish(),
+    );
+
+    const data = await this.sendQuery('/dydxprotocol.affiliates.Query/ReferredBy', requestData);
+
+    return AffiliateModule.ReferredByRequest.decode(data);
```

Comment:
[q] shouldn't this be named `ReferredByResponse`?

Reply:
ohh good catch, derp